bcManager/bcManager.py

```diff
@@ -307,21 +626,294 @@ async def process_match_bcreport(self, ctx, match, tier_md_group_code: str=None,
         await bc_status_msg.edit(embed=embed)
 
         # Step 6: Update match cog info
-        await self.update_match_info(ctx, tier_role.name, match, discovery_data, match_subgroup_json)
+        report = {
+            "winner": discovery_data.get('winner'),
+            "home_wins": discovery_data.get('home_wins'),
+            "away_wins": discovery_data.get('away_wins'),
+            "summary": discovery_data.get('summary'),
+            "ballchasing_id": match_subgroup_json.get('id'),
+            "ballchasing_link": match_subgroup_json.get('link', 
+                f"https://ballchasing.com/group/{match_subgroup_json.get('id')}")
+        }
+        await self.update_match_report(ctx, tier_role.name, match, report)
 
         match_subgroup_json['is_valid_set'] = discovery_data['is_valid_set']
 
         return match_subgroup_json
 
+    async def get_init_score_deep_summary_and_embed(self, ctx, match):
+        title = f"MD {match['matchDay']}: {match['home']} vs {match['away']} [FF Report]"
+        tier_role, match_emoji_url = await self.get_match_tier_role_and_emoji_url(ctx, match)
+        
+        home_franchise_role = (await self.team_manager_cog._roles_for_team(ctx, match['home']))[0]
+        away_franchise_role = (await self.team_manager_cog._roles_for_team(ctx, match['away']))[0]
+        home_emoji = await self.team_manager_cog._get_franchise_emoji(ctx, home_franchise_role)
+        away_emoji = await self.team_manager_cog._get_franchise_emoji(ctx, away_franchise_role)
+        ballchasing_link = match['report']['ballchasing_link']
+
+        bapi : ballchasing.Api = self.ballchasing_api[ctx.guild]
+
+        replays = bapi.get_replays(group_id=match['report']['ballchasing_id'])
+
+        description = "Match Summary\n" + match['report']['summary'] + "\n"
+        embed = discord.Embed(title=title, description=description, color=tier_role.color)
+
+        # previously ff games
+        forfeits = match['report'].get('forfeits')
+        ff_indexes = [ff['game_num'] for ff in forfeits] if forfeits else []
+
+        # valid games
+        r_description = ""
+        game_summaries = []
+        ff_able_reacts = []
+        gi = 1
+        i = 0
+        react_hex_code = 0x1F1E6 # A
+        for replay in replays:
+            while gi in ff_indexes:
+                gi += 1
+                i += 1
+            react_hex = hex(react_hex_code+i)
+            react = self.get_select_reaction(int(react_hex, base=16))
+            ff_able_reacts.append(react)
+            home_goals, away_goals = self.get_home_away_goals(match, replay)
+            winner_emoji = home_emoji if home_goals > away_goals else away_emoji
+            summary = f"{react} **G{gi}:** {match['home']} {home_goals} - {away_goals} {match['away']}"
+            if winner_emoji:
+                summary += f" {winner_emoji}"
+
+            game_summaries.append({
```

Comment:
can you include an example summary string as a comment in this section so we know what it should look like? would help debug if anything goes wrong or if it gets more complex in future

Reply:
yes